app/views/genres/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if current_admin? %>
+  <h1><%= link_to 'Create New Genres Here', new_admin_genre_path %></h1>
```

Comment:
we wanted the form on this page, not to be namespaced and found elsewhere :( User story 2 has a hint that when the visitor views the genres page that they don't see the form